Source/WebCore/css/ComputedStyleExtractor.cpp

```diff
@@ -4452,6 +4462,10 @@ RefPtr<CSSValue> ComputedStyleExtractor::valueForPropertyInStyle(const RenderSty
     case CSSPropertyTextAnchor:
     case CSSPropertyVectorEffect:
         return svgPropertyValue(propertyID);
+
+    // grid-template is handled by ShorthandSeralizer
```

Comment:
To me this comment is confusing to me, and doesn’t feel relevant.

The reason it’s OK to have the `grid-template` case return `nullptr` here is that somehow we know that it’s OK for `valueForPropertyInStyle` and hence for `getPropertyCSSValue` to return `nullptr` for this property. That’s what is non-obvious and thus that is what comment should be about, not about how `getPropertyValue` works.

What is the basis of that? How did we determine that we don’t have any code that tries to get a CSS value for this property?

Reply:
Yeah that's a good point and now is making me rethink whether removing this code is actually correct (doesn't seem like it anymore). Allowing `getPropertyValue` to use `ShorthandSerializer` is one thing but returning `nullptr` here has other implications and probably very unexpected for other callers